### PR TITLE
Yessenia Rodriguez express-post-json

### PR DESCRIPTION
**Pull request by yesseniarod**
Screen Shot 2021-03-16 at 3 40 25 PM

### CODE REVIEW

File: express-post-json/index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+});
+
+const data = express.json();
+app.use(data);
+
+app.post('/api/grades', (req, res) => {
+  req.body.id = nextId;
+  res.send(req.body);
+  res.status(201);
+  grades[0] = req.body;
```
Comment (thebearingedge) [inline express-post-json/index.js:26]: `[0]` This is what the `nextId` is for, like `node-notes-cli`.

File: express-post-json/index.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  console.log('Express server listening on port 3000');
+});
+
+const data = express.json();
+app.use(data);
+
+app.post('/api/grades', (req, res) => {
+  req.body.id = nextId;
+  res.send(req.body);
+  res.status(201);
```
Comment (thebearingedge) [inline express-post-json/index.js:25]: These should be reversed so that the status code is `201` instead of `200`. Check your response.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Ok, it's getting close. Please see my notes.

File: express-post-json/index.js (hunk 2 above)
Comment (yesseniarod) [reply express-post-json/index.js]: Screen Shot 2021-03-16 at 3 51 26 PM

File: express-post-json/index.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+});
+
+const data = express.json();
+app.use(data);
+
+app.post('/api/grades', (req, res) => {
+  req.body.id = nextId;
+  res.status(201);
+  res.send(req.body);
+  grades[nextId] = req.body;
```
Comment (thebearingedge) [inline express-post-json/index.js:26]: Be sure to also increment `nextId` or else every added object will replace the last one.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**